daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -294,6 +297,10 @@ message Type {
     Fun fun = 4;
     Forall forall = 5;
     Tuple tuple = 7;
+    // *Available since version 1.dev*
+    // *Must be positive*
+    // use signed integer for future use.
+    sint64 nat = 11;
```

Comment:
Isn't there a `uint64`?

Reply:
I prefer to use a signed version, as we may generalize the type parameter `nat` in the future. For instance, decimal with negative scale makes perfect sens.